### PR TITLE
ENH: Add unpack-command

### LINKED ISSUE #12: pyenvbuilder unpack environment from a tarball

### PR DESCRIPTION
**Pull request by cristinasewell**
Added the `unpack` command.
Also, added an else statement with an error log to the `pack.py`  - I realized I needed this after implemented the `unpack-commnd`

Closes #12

### CODE REVIEW

File: pyenvbuilder/commands/unpack.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        self.name = 'unpack'
+        self.help = 'Unpacks an environment from a tarball'
+
+    def run(self, **kwargs):
+        envs = kwargs.get('tarballs')
+        for e in envs:
+            if validate_path(e)[0]:
+                self.unpack(e)
+            else:
+                logger.error(f'When unpacking could not validate {e}')
```
Comment (hhslepicka) [inline pyenvbuilder/commands/unpack.py]: `could not validate path`...

File: pyenvbuilder/commands/unpack.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            'tar -xzf $env_name.tar.gz -C $env_name\n'
+            'source $env_name/bin/activate\n'
+            'conda-unpack\n'
+            'source $env_name/bin/deactivate\n')
+
+        unpack_arguments = unpack_template.substitute(
+            env_name=env)
+
+        unpack_proc = run_subprocess(unpack_arguments)
+        logger.info(f'Pack subprocess return code: {unpack_proc.returncode}')
```
Comment (hhslepicka) [inline pyenvbuilder/commands/unpack.py]: Unpack subprocess...

File: pyenvbuilder/commands/unpack.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        Parameters
+        ----------
+        env: string
+            Environment.gz name/path
+        '''
+        env = env.split('.tar.gz')[0]
+
+        unpack_template = Template(
+            'mkdir $env_name\n'
+            'tar -xzf $env_name.tar.gz -C $env_name\n'
```
Comment (hhslepicka) [inline pyenvbuilder/commands/unpack.py:44]: I would just suggest to maybe add a `echo Unpacking the environment tarball...` so users have some feedback here.

**Review by hhslepicka — CHANGES_REQUESTED** [review]
Just some minor change requests.

File: pyenvbuilder/commands/unpack.py (hunk 1 above)
Comment (cristinasewell) [reply pyenvbuilder/commands/unpack.py]: got it, thank you

File: pyenvbuilder/commands/unpack.py (hunk 2 above)
Comment (cristinasewell) [reply pyenvbuilder/commands/unpack.py]: yep, copy and paste issue....

File: pyenvbuilder/commands/unpack.py (hunk 3 above)
Comment (cristinasewell) [reply pyenvbuilder/commands/unpack.py:44]: Yeah this is a good idea, thank you. I added it now

**Review by hhslepicka — APPROVED** [review]

**Pull request merged**